packages/samples/headless-ssr-commerce/lib/commerce-engine-config.ts

```diff
@@ -31,14 +31,6 @@ type CommerceEngineConfig = CommerceEngineDefinitionOptions<
 export default {
   configuration: {
     ...getSampleCommerceEngineConfiguration(),
-    context: {
-      language: 'en',
-      country: 'US',
-      currency: 'USD',
-      view: {
-        url: 'https://sports.barca.group/browse/promotions/ui-kit-testing',
-      },
-    },
```

Comment:
Should we remove this from the require definition since it should always be added during fetchStaticState ?